### ISSUE TITLE
Block editor: Wide width blocks inside a group don’t have vertical margins inside the editor

### ISSUE THREAD

**Issue by briceduclos** [WordPress/twentytwentyone#717]
**Describe the bug**
Inside the editor, blocks that are set to wide width don’t have any top and bottom margins when placed inside a group. As a result, there’s no spacing between two consecutive wide width blocks inside a group. 

**To Reproduce**
Inside the code editor, paste the following code. It consists of a group containing 2 cover blocks set to wide width (this example uses the Cover block but it can be replaced by any wide block: image, media & text; group…).

```
<!-- wp:group {"align":"full","backgroundColor":"gray"} -->
<div class="wp-block-group alignfull has-gray-background-color has-background"><div class="wp-block-group__inner-container"><!-- wp:cover {"overlayColor":"purple","contentPosition":"center center","align":"wide"} -->
<div class="wp-block-cover alignwide has-purple-background-color has-background-dim"><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","fontSize":"large"} -->
<p class="has-text-align-center has-large-font-size">Wide Cover Block</p>
<!-- /wp:paragraph --></div></div>
<!-- /wp:cover -->

<!-- wp:cover {"overlayColor":"green","align":"wide"} -->
<div class="wp-block-cover alignwide has-green-background-color has-background-dim"><div class="wp-block-cover__inner-container"><!-- wp:paragraph {"align":"center","fontSize":"large"} -->
<p class="has-text-align-center has-large-font-size">Wide Cover Block</p>
<!-- /wp:paragraph --></div></div>
<!-- /wp:cover --></div></div>
<!-- /wp:group -->
```

**Screenshots**
| Editor  | Front |
| ------------- | ------------- |
| ![wide-blocks-group](https://user-images.githubusercontent.com/56188815/97374268-39573e00-18b8-11eb-92a7-9c2ca6dc570e.PNG)  | ![wide-blocks-group-front](https://user-images.githubusercontent.com/56188815/97374265-38bea780-18b8-11eb-8d38-61280ea0e0b6.PNG)  |

**Comment by carolinan** [comment]
My question is if we want the spacing between the inner blocks at all?

**Comment by ryelle** [comment]
I think this will be fixed with #723, and by "fixed" i mean the editor will gain the margin between the two blocks.
